### ISSUE TITLE
Documentation: python-dev, build-essential needed to install

### ISSUE THREAD

**Issue by rillip3** [cloudnull/turbolift#11]
tl;dr install python-dev and build-essential (or groupinstall "Development Tools" for RHEL/Cent) or you'll have a bad time. It'd be nice to have that in the requirements.

long version:
I had to bang my head on this for a while, but hashlib is needed to run turbolift. At first it would fail saying it couldn't find gcc to install hashlib - duh, install build-essential. done.  But then I got a more generic gcc failed error.  I dug at it for a while and it turns out Python.h is searched for by GCC as part of installing hashlib.

I installed build-essential and python-dev, and then started getting weird errors about pytz when trying to run turbolift. I realized that there was more stuff left to do between hashlib and the end of the setup, ran setup.py install again, now turbolift installs fine
